client/src/partials/enterprise/enterprise.html

```diff
@@ -30,174 +33,217 @@
-          </fieldset>
-
-          <input type="submit" class="btn btn-sm btn-success" ng-click="saveEdit()" value="{{ 'FORM.SAVE_CHANGES' | translate }}" ng-disabled="editForm.$invalid">
-          <input type="button" class="btn btn-sm btn-default" ng-click="resetEdit()" value="{{ 'FORM.RESET_FORM' | translate }}">
-
-        </form>
+        <div class="panel panel-primary" ng-switch-when="update">
+          <div class="panel-heading">{{ "ENTERPRISE.EDIT_ENTERPRISE" | translate }}</div>
+          <div class="panel-body">
+            <form class="form-horizontal" name="UpdateForm" ng-submit="EnterpriseCtrl.submit(UpdateForm.$invalid)" novalidate>
+
+              <div class="form-group" ng-class="{ 'has-error' : UpdateForm.$submitted && UpdateForm.name.$invalid }">
+                <label class="col-xs-3 control-label required" for="bhima-enterprise-name">{{ 'ENTERPRISE.NAME' | translate }}</label>
+                <div class="col-xs-9">
+                  <input type="text" class="form-control" name="name" id="bhima-enterprise-name" ng-model="EnterpriseCtrl.enterprise.name" required>
+                </div>
+              </div>
+
+              <div class="form-group" ng-class="{ 'has-error' : UpdateForm.$submitted && UpdateForm.abbr.$invalid }">
+                <label for="bhima-enterprise-abbr" class="col-md-3 control-label">{{ "ENTERPRISE.ABBREVIATION" | translate }}</label>
+                <div class="col-md-9">
+                  <input type="text" class="form-control" name="abbr" required id="bhima-enterprise-abbr" ng-model="EnterpriseCtrl.enterprise.abbr">
+                </div>
+              </div>
+
+              <div class="form-group">
+                <label for="bhima-enterprise-po_box" class="col-md-3 control-label">{{ "ENTERPRISE.PO_BOX" | translate }}</label>
+                <div class="col-md-9">
+                  <input type="text" class="form-control" name="po_box" id="bhima-enterprise-po_box" ng-model="EnterpriseCtrl.enterprise.po_box">
+                </div>
+              </div>
+
+              <div class="form-group">
+                <label for="bhima-enterprise-email" class="col-md-3 control-label">{{ "ENTERPRISE.EMAIL" | translate }}</label>
+                <div class="col-md-9">
+                  <input type="email" class="form-control" name="email" id="bhima-enterprise-email" ng-model="EnterpriseCtrl.enterprise.email">
+                </div>
+              </div>
+
+              <div class="form-group">
+                <label for="bhima-enterprise-phone" class="col-md-3 control-label">{{ "ENTERPRISE.PHONE" | translate }}</label>
+                <div class="col-md-9">
+                  <input type="tel" class="form-control" name="phone" id="bhima-enterprise-phone" ng-model="EnterpriseCtrl.enterprise.phone">
+                </div>
+              </div>
+
+              <div class="form-group" ng-class="{ 'has-error' : UpdateForm.$submitted && UpdateForm.location_id.$invalid }">
```

Comment:
These locations (see line 138 as well) should probably use the [location select directive](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/js/directives/locationSelect.js)